runtime_config_linux.go

```diff
@@ -39,8 +39,10 @@ type LinuxRuntime struct {
 	SelinuxProcessLabel string `json:"selinuxProcessLabel"`
 	// Seccomp specifies the seccomp security settings for the container.
 	Seccomp Seccomp `json:"seccomp"`
-	// RootfsPropagation is the rootfs mount propagation mode for the container
+	// RootfsPropagation is the rootfs mount propagation mode for the container.
 	RootfsPropagation string `json:"rootfsPropagation,omitempty"`
+	// disableNewPrivileges controls whether additional privileges could be gained by processes in the container.
+	DisableNewPrivileges `json:"disableNewPrivileges,omitempty"`
```

Comment:
Missing `*bool`?  At some point I think we should land general specs for the:

> [Null and missing entries are valid JSON and mean “don't touch this”.][1]

semantics .  If we're not comfortable landing that generally, I think this PR needs docs about what runtimes should do if `DisableNewPrivileges` is null or unset in the JSON.

[1]: https://github.com/opencontainers/specs/pull/233#issuecomment-152606572